backend/substrapp/tasks/tasks.py

```diff
@@ -797,9 +805,38 @@ def _do_task(client, subtuple_directory, tuple_type, subtuple, compute_plan_id,
     with open(path.join(pred_path, 'perf.json'), 'r') as perf_file:
         perf = json.load(perf_file)
     result['global_perf'] = perf['all']
+
+    tag = subtuple.get("tag")
+    if tag and TAG_VALUE_FOR_TRANSFERT_BUCKET in tag:
+        transfer_to_bucket(subtuple["key"], [pred_path, model_path])
+
     return result
 
 
+def transfer_to_bucket(tuple_key, paths):
+    if not ACCESS_KEY or not SECRET_KEY or not BUCKET_NAME:
+        logger.info(f'unset global env for bucket transter: {ACCESS_KEY} {SECRET_KEY} {BUCKET_NAME}')
+        return
+
+    tar_file = f'{tuple_key}.tar.gz'
+    tar_path = path.join('/tmp', tar_file)
+
+    try:
+        with tarfile.open(tar_path, 'x:gz') as tar:
+            for file_path in paths:
```

Comment:
`dir_path` is more appropriate for this loop

Reply:
Done